### PR TITLE
new branch modified

### PR DESCRIPTION
**Pull request by Basma-Abdallah**

### CODE REVIEW

**Review by Nora-Lootfy — CHANGES_REQUESTED** [review]
You have done it the hard way,, try to search how to do it with Laravel it is much simpler but your code still do the job,, good job ya Basma

File: app/Http/Controllers/ClassController.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+    
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(Request $request)
+    {
+       //dd($request);
+      $className=$request->className;
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/ClassController.php]: try using associative array rather than lots of variables,, it is more cleaner

**Review by Nora-Lootfy — APPROVED** [review]
Good job ya Basma, you have nailed it

File: app/Http/Controllers/ClassController.php [hunk 2]
```diff
@@ -82,14 +83,35 @@ public function edit(string $id)
...
     /**
      * Remove the specified resource from storage.
      */
-    public function destroy(string $id)
+    public function destroy(Request $request)
     {
-        //
+    $id = $request->id;
+    Classes::where('id', $id)->delete();
+    return redirect('classes');
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/ClassController.php:138]: you have forget to add trashed classes function and file

**Review by Nora-Lootfy — CHANGES_REQUESTED** [review]
good job ya Basma

File: app/Http/Controllers/ClassController.php [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+        //   //$is_fulled= $request->fulled;
+        //   $is_fulled= isset($request->fulled);
+        //   $timeFrom= $request->time_from;
+        //   $timeTo= $request->time_to;
+
+      $data= $request->validate([
+        'className'=>'required|string',
+         'capacity'=>'required|numeric|min:2|max:25',
+         'price'=>'required|decimal:0,2',
+         'timeFrom'=>'required|date_format:H:i',
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/ClassController.php:46]: check that timeFrom is before timeTo

**Review by Nora-Lootfy — APPROVED** [review]
Close the pull request ya Basma, after i review it then open a new on after every task submitting

Have a nice day😊

File: app/Http/Controllers/ClassController.php (hunk 3 above)
Comment (Basma-Abdallah) [reply app/Http/Controllers/ClassController.php:46]: 'strtotime($class->timeFrom)'=>'date_format:H:i',
             'strtotime($class->timeTo)'=>'date_format:H:i|after:timeFrom'
        

i did code like this but I don't know why these changes didn't appear in the last commit

**Pull request merged**